token/js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@solana/spl-token",
-  "version": "0.1.8",
+  "version": "0.1.9",
```

Comment:
We'll take care of bumping the version prior to deployment, so there's no need to do that here.